Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -246,7 +246,7 @@ protected virtual bool OnShouldReturn(UITextField view)
 			Control.ResignFirstResponder();
 			((IEntryController)Element).SendCompleted();
```

Comment:
If the goal is to fix a NRE when firing the completed event, shouldn't you be null-checking on/before the line where the Completed event is actually fired?